MANIFEST.in

```diff
@@ -2,7 +2,15 @@ global-exclude .git*
 include LICENSE.txt
 include versioneer.py
 recursive-include examples *.py *.ipynb *.md
-graft bokehjs
-prune bokehjs/node_modules
-prune examples/plotting/notebook/.ipynb_checkpoints
+prune examples/charts/.ipynb_checkpoints
+prune examples/compat/mpl/.ipynb_checkpoints
 prune examples/glyphs/.ipynb_checkpoints
+prune examples/plotting/notebook/.ipynb_checkpoints
+prune examples/charts/*.html
+prune examples/compat/ggplot/*.html
+prune examples/compat/mpl/*.html
+prune examples/compat/pandas/*.html
+prune examples/compat/seaborn/*.html
+prune examples/embed/*.html
+prune examples/glyphs/*.html
+prune examples/plotting/file/*.html
```

Comment:
maybe `recursive-exclude` could be used to simplify this?

https://docs.python.org/2/distutils/sourcedist.html#commands